Week3/homework/homework.sql

```diff
@@ -0,0 +1,160 @@
+-- MySQL dump 10.13  Distrib 5.7.21, for Linux (x86_64)
+--
+-- Host: localhost    Database: company
+-- ------------------------------------------------------
+-- Server version	5.7.21-0ubuntu0.16.04.1
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `Departement`
+--
+
+DROP TABLE IF EXISTS `Departement`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `Departement` (
+  `Dept_Number` int(11) NOT NULL AUTO_INCREMENT,
+  `Dept_Name` varchar(50) NOT NULL,
+  `Dept_Location` varchar(50) DEFAULT NULL,
+  PRIMARY KEY (`Dept_Number`),
+  UNIQUE KEY `Dept_Name` (`Dept_Name`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `Departement`
+--
+
+LOCK TABLES `Departement` WRITE;
+/*!40000 ALTER TABLE `Departement` DISABLE KEYS */;
+/*!40000 ALTER TABLE `Departement` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `Emp_Info`
+--
+
+DROP TABLE IF EXISTS `Emp_Info`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `Emp_Info` (
+  `Emp_Number` int(11) NOT NULL,
+  `Emp_Phone` varchar(50) NOT NULL,
+  `Emp_Email` varchar(50) NOT NULL,
+  `Emp_Addres` varchar(50) NOT NULL,
+  KEY `Emp_Number` (`Emp_Number`),
+  CONSTRAINT `Emp_Info_ibfk_1` FOREIGN KEY (`Emp_Number`) REFERENCES `Employee` (`Emp_Number`)
```

Comment:
Is there a 1-many, many-to-many, or 1-1 relationship between Emp_Info and Employee?

I would have thought that this was 1-1, in which case these could have been attributes of the Employee table.